Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHG_cff.py

```diff
@@ -0,0 +1,77 @@
+import FWCore.ParameterSet.Config as cms
+
+# ------------------------------------------------------------------------------
+# configure a filter to run only on the events selected by TkAlMinBias AlcaReco
+from Alignment.CommonAlignmentProducer.ALCARECOPromptCalibProdSiPixelAli_cff import *
+ALCARECOTkAlMinBiasFilterForSiPixelAliHG = ALCARECOTkAlMinBiasFilterForSiPixelAli.clone()
+
+
+from Alignment.CommonAlignmentProducer.LSNumberFilter_cfi import *
+
+# Ingredient: offlineBeamSpot
+from RecoVertex.BeamSpotProducer.BeamSpot_cfi import offlineBeamSpot
+
+# Ingredient: AlignmentTrackSelector
+# track selector for HighPurity tracks
+#-- AlignmentTrackSelector
+SiPixelAliHighPuritySelectorHG = SiPixelAliHighPuritySelector.clone()
+
+# track selection for alignment
+SiPixelAliTrackSelectorHG = SiPixelAliTrackSelector.clone(
+	src = 'SiPixelAliTrackFitterHG'
+)
+
+# Ingredient: SiPixelAliTrackRefitter0
+# refitting
+SiPixelAliTrackRefitterHG0 = SiPixelAliTrackRefitter0.clone(
+	src = 'SiPixelAliHighPuritySelectorHG'
+)
+SiPixelAliTrackRefitterHG1 = SiPixelAliTrackRefitterHG0.clone(
+	src = 'SiPixelAliTrackSelectorHG'
+)
+
+#-- Alignment producer
+SiPixelAliMilleAlignmentProducerHG = SiPixelAliMilleAlignmentProducer.clone()
+SiPixelAliMilleAlignmentProducerHG.ParameterBuilder.Selector = cms.PSet(
+    alignParams = cms.vstring(
+	"TrackerP1PXBLadder,111111",
+	"TrackerP1PXECPanel,111111",
+        )
+    )
+SiPixelAliMilleAlignmentProducerHG.tjTkAssociationMapTag = 'SiPixelAliTrackRefitterHG1'
+SiPixelAliMilleAlignmentProducerHG.algoConfig = MillePedeAlignmentAlgorithm.clone()
+SiPixelAliMilleAlignmentProducerHG.algoConfig.binaryFile = 'milleBinaryHG_0.dat'
```

Comment:
any reason why the parameters cannot be changed directly within the clone function itself?

Reply:
I thought I could do it like this:
```Python
SiPixelAliMilleAlignmentProducerHG = SiPixelAliMilleAlignmentProducer.clone(
    ParameterBuilder.Selector = cms.PSet(
      alignParams = cms.vstring(
	"TrackerP1PXBLadder,111111",
	"TrackerP1PXECPanel,111111",
      )
    ),
    tjTkAssociationMapTag = 'SiPixelAliTrackRefitterHG1',
    algoConfig = MillePedeAlignmentAlgorithm.clone(
	binaryFile = 'milleBinaryHG_0.dat',
	treeFile = 'treeFileHG.root',
	monitorFile = 'millePedeMonitorHG.root'
    )
)
```
But when compiling I get:
```
*** Error compiling 'src/Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHG_cff.py'...
  File "src/Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHG_cff.py", line 35
    ParameterBuilder.Selector = cms.PSet(
    ^
SyntaxError: expression cannot contain assignment, perhaps you meant "=="?
```

Reply:
i think you might want to use `dict` instead.